cmake/modules/RootBuildOptions.cmake

```diff
@@ -181,10 +177,6 @@ option(testing "Enable testing with CTest" OFF)
 
 set(gcctoolchain "" CACHE PATH "Set path to GCC toolchain used to build llvm/clang")
 
-if (runtime_cxxmodules)
```

Comment:
@Axel-Naumann I thought we would like to have possibility to disable pch for runtime cxxmodules? @vgvassilev

Reply:
*Indeed*!!! Dammit, sorry this slipped through. Thanks for pointing that out.

Guilherme, can you revert that part, please?

Reply:
It used to be the case that the generation of the pch used the value of the `pch` option. This is no longer true, only the value of the modules build is used now. For example, see [here](https://github.com/root-project/root/blob/master/CMakeLists.txt#L384-L407). The PCH is now in an `else()` part of `if(runtime_cxxmodules)`.

Reply:
Ok I see.. Thanks! maybe it will make more sense (in the future) actually to enable this option properly back? So we will not have this tricky else() branch...

Reply:
That would allow people to use `-Druntime_cxxmodules=ON -Dpch=ON`, which will not work.